$MirOS: ports/devel/cvs/patches/patch-src_modules_c,v 1.1 2006/10/02 05:59:17 tg Exp $
--- src/modules.c.orig	Sat Mar 10 23:14:20 2007
+++ src/modules.c	Sat Mar 10 23:14:14 2007
@@ -29,6 +29,7 @@
 #include "cvs.h"
 #include "save-cwd.h"
 
+__RCSID("$MirOS$");
 
 /* Defines related to the syntax of the modules file.  */
 
@@ -450,7 +451,7 @@ my_module (DBM *db, char *mname, enum mt
 	    case '?':
 		error (0, 0,
 		       "modules file has invalid option for key %s value %s",
-		       key.dptr, value);
+		       (char *)key.dptr, value);
 		err++;
 		goto do_module_return;
 	}
